{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40792115",
  "multipleFaultNumbers": null,
  "date": "2020-12-29T15:14:51",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-29T15:14:51",
  "information": "",
  "actualTimeOfRestoration": "2020-12-29T15:14:51",
  "outageCentrePoint": {
    "lat": 53.552881,
    "lng": -2.524334
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL5 3GJ, BL5 3SA, BL5 3SF, BL5 3SN, BL5 3SQ, BL5 3SW",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GS"
}